src/frontend/src/views/Data/headers.tsx

```diff
@@ -152,8 +151,8 @@ export const TREE_HEADERS: Header[] = [
     },
   },
   {
-    key: "downloadLink",
-    sortKey: ["downloadLink"],
+    key: "actionMenu",
+    sortKey: ["actionMenu"],
```

Comment:
whoa, I just realized they can sort the download/action column. That's... weird. If we just false-y this, can we kill this behavior? Sorting by all identical items is wacky, but if it's not super easy to achieve because of how the whole table rendering is setup, oh well